.travis.yml

```diff
@@ -0,0 +1,30 @@
+sudo: false
+
+language: go
+
+go:
+  - 1.9
+  - master
+
+matrix:
+  # Allow failures if the code fails on unstable development versions of Go.
+  allow_failures:
+    - go: master
+  
+  # Don't block on tip tests. Mark the build as green if tests pass on stable versions.
+  fast_finish: true
+
+# Skip install. Only build with the code in vendor; no `go get`.
+install: true
+
+before_script:
+  - GO_FILES=$(find . -iname '*.go' -type f | grep -v /vendor/)
+
+script:
+  - go build -v
```

Comment:
Can we do docker build instead?

Reply:
Yeah, we can build *and* push. I can add in `docker build` for this PR and we can talk about `push` separately since we're using Docker Hub already.

Reply:
Ideally we want to trigger `docker build` on PR creation and updates. `docker push` only occurs when merging back to master

Reply:
sure, please create another issue for it. Meanwhile you could :shipit: